content/1.7/back-office/dashboard/dashboard-configure.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Dashboard Configure
+weight: 1
+---
+
+# Dashboard configure
+
+3 blocks are displayed on this page:
+
+ - **Average bank fees per payment method**
+ - **Average shipping fees per shipping method**
+ - **Other settings**
+
+## Demo mode
+
+ - **Demo mode**: This toggle allows to enable or disable fake data. By default, it is disabled. When enabled, fake data are used. When disabled, the real store's data are used. The store's data are gathered by PrestaShop's statistics system.
+The following tooltip is displayed: "This mode displays sample data so you can try your dashboard without real numbers."
+
+## Average bank fees per payment method
+
+4 default fields are displayed in this block, if more payment methods are added to the shop, they will be displayed additionally here:
+
+ - **Bank transfer** - An help text is displayed under the field: "Choose a fixed fee for each order placed in USD% with Bank transfer." Only numbers are allowed in this field. On the right of this field, the default currency (defined in International > Localization) is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Bank transfer field is invalid."
+ - **Bank transfer** - An help text is displayed under the field: "Choose a variable fee for each order placed in USD% with Bank transfer. It will be applied on the total paid with taxes." Only numbers are allowed in this field. On the right of this field, a percentage sign is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Bank transfer field is invalid.".
+ - **Payments by check** - An help text is displayed under the field: "Choose a fixed fee for each order placed in USD% with Payments by check.". Only numbers are allowed in this field. On the right of this field, the default currency (defined in International > Localization) is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Payments by check field is invalid."
+ - **Payments by check** - An help text is displayed under the field: "Choose a variable fee for each order placed in USD% with Payments by check. It will be applied on the total paid with taxes.". Only numbers are allowed in this field. On the right of the field, a percentage sign is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Payments by check field is invalid."
+
+
+### Call to action
+
+ - **Save** - When pressed and if there is no error, the following success message is displayed: "The settings have been successfully updated." and the form is saved.
+
+When the multistore is enabled, there are checkboxes before each field of this page. The shop association block is displayed when the multistore is enabled and if several shops are configured.
```

Comment:
Could you move the paragraph related to the multistore at the end of this file in a new "## Multistore" section ?